build.fsx

```diff
@@ -52,573 +50,347 @@ module MessagePrompts =
+module DocumentationTasks =
 
-let publishPrereleaseNugetPackages = 
-    BuildTask.create "publishPrereleaseNugetPackages" [clean; buildAll; runTestsAll; buildPrereleasePackages] {
-        Paket.push(fun p ->
-            { p with
-                WorkingDir = pkgDir
-                ToolType = ToolType.CreateLocalTool()
-                ApiKey = Environment.environVarOrDefault "NuGet-key" "" })
-    }
+    open ProjectInfo
 
-// --------------------------------------------------------------------------------------
-// Generate the documentation
+    open BasicTasks
 
-let generateDocumentation = 
-    BuildTask.create "generateDocumentation" [cleanDocs; buildAll.IfNeeded] {
-        let result =
-            DotNet.exec
-                id
-                "fsdocs"
-                (sprintf "build --eval --property Configuration=%s --noapidocs" configuration)
+    let initDocPage = BuildTask.create "InitDocsPage" [] {
+        printfn "Please enter filename"
+        let filename = System.Console.ReadLine()
         
-        if not result.OK then 
-            failwith "error generating docs" 
+        printfn "Please enter title"
+        let title = System.Console.ReadLine()
+
+        let path = "./docs" </> filename
+
+        let lines = """(*** hide ***)
+(*** condition: prepare ***)
+#r "../bin/ISADotNet/netstandard2.0/ISADotNet.dll"
+#r "../bin/ISADotNet.XLSX/netstandard2.0/ISADotNet.XLSX.dll"
+(*** condition: ipynb ***)
+#if IPYNB
+#r "nuget: Plotly.NET, {{fsdocs-package-version}}"
+#r "nuget: Plotly.NET.Interactive, {{fsdocs-package-version}}"
+#endif // IPYNB
+(**
+# {{TITLE}}
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/plotly/Plotly.NET/gh-pages?filepath={{FILENAME}}.ipynb)
+*)
+"""
+
+        if (promptYesNo (sprintf "creating file %s with title %s OK?" path title)) then
+            lines
+            |> String.replace "{{FILENAME}}" filename
+            |> String.replace "{{TITLE}}" title
+            |> fun content -> File.WriteAllText (path,content)
+        else
+            failwith "aborted"
     }
 
-let crackDocumentationInlineItems = 
-    BuildTask.create "crackDocumentationInlineItems" [generateDocumentation] {
-        let tempDocsDir = "output/content"
-        Shell.replaceInFiles 
-            (seq {
-                yield "<table class=\"pre\"><tr><td><pre><code>\"&lt;inlineHTML&gt;",""
-                yield "&lt;/inlineHTML&gt;\"</code></pre></td></tr></table>",""
-                yield "&lt;","<"
-                yield "&gt;",">"
-            }) 
-            (Directory.EnumerateFiles tempDocsDir |> Seq.filter (fun x -> x.EndsWith(".html")))
+    let buildDocs = BuildTask.create "BuildDocs" [build; copyBinaries] {
+        printfn "building docs with stable version %s" stableVersionTag
+        runDotNet 
+            (sprintf "fsdocs build --eval --clean --property Configuration=Release --parameters fsdocs-package-version %s" stableVersionTag)
```

Comment:
Related to: #11 
I think atm it would be good to add the `--noapidocs` flag until this API documentantion generation issue is resolved in `FSharp.Formatting`